tests/frameworks/OSXShims/include/windows.h

```diff
@@ -0,0 +1,55 @@
+//******************************************************************************
```

Comment:
[https://github.com/Microsoft/WinObjC/issues/687](https://github.com/Microsoft/WinObjC/issues/687)

---
In reply to: [72351063](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72351063) [](ancestors = 72351063,72312429)